doorkeeper-openid_connect.gemspec

```diff
@@ -24,11 +24,14 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.7'
 
-  spec.add_runtime_dependency 'doorkeeper', '>= 5.5', '< 5.8'
+  spec.add_runtime_dependency 'doorkeeper', '>= 5.5', '< 5.9'
   spec.add_runtime_dependency 'jwt', '>= 2.5'
 
+  spec.add_development_dependency 'bigdecimal'
   spec.add_development_dependency 'conventional-changelog', '~> 1.2'
+  spec.add_development_dependency 'drb'
   spec.add_development_dependency 'factory_bot'
+  spec.add_development_dependency 'mutex_m'
```

Comment:
Why do we need `mutex_m` and `bigdecimal`? :thinking:

Reply:
@nbulaj Ruby 3.4 moved these gems out: https://docs.ruby-lang.org/en/master/NEWS_md.html

Reply:
Wow, didn't know! Thanks :bow: